rcon/server_stats.py

```diff
@@ -67,14 +67,6 @@ def save_server_stats_for_last_hours(hours=24, skip_last_hours=1):
     return save_server_stats_for_range(start, end)
 
 
-def save_server_stats_since_inception():
```

Comment:
Why not just leave this one so it doesn't need to get rewritten later? With the CLI disabled and it not being called in `entrypoint.sh` it shouldn't hurt anything.